### ISSUE TITLE
wrong userpic in timeline when they have the same filename

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#30]
```
Assuming two different users have each uploaded a userpic with the same
name (e.g. "avatar.png") but with different contents.


Pino is expected to display the two different userpics appropriate. Instead
it shows the same picture for both users (apparently the one it loaded
earlier).


Tested on pino 0.1.1 / Ubuntu 9.10

```

Original issue reported on code.google.com by `m...@nilsbreyer.eu` on 3 Jan 2010 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
Oh, this is so stupid mistake! Sorry, I will fix it soon.
```

Original comment by `tro...@gmail.com` on 3 Jan 2010 at 8:35
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 45f4d6ba23 revision.
```

Original comment by `tro...@gmail.com` on 4 Jan 2010 at 1:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you for fixing it as fast! 


```

Original comment by `m...@nilsbreyer.eu` on 4 Jan 2010 at 3:49
